### PR TITLE
chore(functions): remove unused samples

### PR DESCRIPTION
**Pull request by ace-n**
This PR removes the following directories that are now unused:
- `functions/tips`
- `functions/concepts`

Fixes b/257346317

### CODE REVIEW

**Comment by ace-n** [comment]
**Note:** these samples now live elsewhere (see #8554 and #8555).

I'm **assuming** the _Sample Browser_ alerts from `snippet-bot` are a non-issue.

**Review by nicain — APPROVED** [review]

File: functions/concepts/README.md [hunk 1]
```diff
@@ -1,11 +0,0 @@
-<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
```
Comment (averikitsch) [inline functions/concepts/README.md:1]: Do we want to keep these READMEs?

Comment (ace-n) [reply functions/concepts/README.md:1]: Will address in subsequent PR. 👍

**Comment by ace-n** [comment]
- `snippet-bot` errors are expected

➡️ time to **admin merge**

**Pull request merged**